test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/KeyValuePairModelBinderTest.cs

```diff
@@ -309,7 +309,9 @@ private static IModelBinder CreateIntBinder()
                 {
                     if (mbc.ModelType == typeof(int))
                     {
-                        return Task.FromResult(new ModelBindingResult(42, mbc.ModelName, true));
+                        var model = 42;
+                        var validationNode = new ModelValidationNode(mbc.ModelName, mbc.ModelMetadata, model);
+                        return Task.FromResult(new ModelBindingResult(model, mbc.ModelName, true, validationNode));
```

Comment:
(it's test code but ...) probably worth sticking in the `isModelSet` argument name here and a couple of places below